data/templated/example.xml

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!-- Note: these example templates reuse TPC-C queries and data as a demonstration, but others are possible. -->
+<!-- TODO: Add additional parameter data types to increase test coverage. -->
```

Comment:
Especially NULLs

Reply:
Did this for explicit NULL types, but not "NULLable" types (e.g., `c VARCHAR NULL` type columns).
The issue here is that we convert params to CSVs lists when passing them to objects which changes those values into "empty" strings instead.

Reply:
Let's take this up in future work.